discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -253,6 +253,11 @@ def validate_track_tx(params: ManageEntityParameters):
                 f"Existing track {track_id} does not match user"
             )
 
+        if params.action == Action.UPDATE and not existing_track.is_unlisted and params.metadata.get("is_unlisted"):
+            raise IndexingValidationError(
+                f"Cannot unlist track {track_id}"
+            )
```

Comment:
nit: I think this would help if we see this attempted a bunch in logs
```suggestion
            raise IndexingValidationError(
                f"Cannot unlist already listed track {track_id}"
            )
```